src/test/unit/mocks/subscriptionCaseList.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
this is a hearing, we already have hearings

Reply:
removed the file and now using hearings mock file.